docs/Reference/API-Methods.md

```diff
@@ -4390,6 +4390,109 @@ None
     methods, use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api) or
     [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options.
 
+### txpool_besuPendingTransactions
+
+Lists pending transactions that match the supplied filter conditions.
+
+
+#### Parameters
+
+* `QUANTITY` - Integer representing the maximum number of results to return.
+* Object containing the filter conditions. Filter pending transactions based on the following fields
+    and supported operators:
```

Comment:
I think some fo the filter conditions need to be explicitly spelled out.
```suggestion
* Object containing the filter conditions. 
  Each field in the object corresponds to a field, and the value is a single field object where the field name is the operator and the field value is the value for the operator.  Multiple transactions fields can be provided, and all filters must be satisfied for a transaction to be returned.  Each field can have only one operator (i.e. you cannot ask for transactions whose gas price is between 8 and 9 GWei, by using both the `gt` and `lt` operator).
```

Reply:
@shemnon can we supply the same field twice? For example:

```
             "gas":{
                "lt":"0x5209"
             },
             "gas":{
                "gt":"0x4000"
             }
```

Reply:
Actually, ignore^^, I just tested it with the following and it worked fine.

```
{
   "jsonrpc":"2.0",
   "method":"txpool_besuPendingTransactions",
   "params":[
      2,
      {
         "gas":{
            "lt":"0x5209"
         },
         "gas":{
            "gt":"0x4209"
         },
         "nonce":{
            "gt":"0x1"
         }
      }
   ],
   "id":1
}
```

Reply:
I've reworded you suggestion slightly.

Reply:
That shouldn't work.  I would expect that only one of the "gas" values will have an effect, since it goes into a hashmap.

Reply:
@shemnon, should we have some sort of check for this? i.e Duplicate field names.

Reply:
It's actually bad JSON.  https://tools.ietf.org/html/rfc8259#section-4 

>The names within an object SHOULD be unique.

The particular details of our Library obscures this from us before we can even get a chance to detect it.  It puts the fields in a hash map.